bundles/AdminBundle/public/js/pimcore/element/selector/quickSearchFacade.js

```diff
@@ -0,0 +1,42 @@
+pimcore.registerNS('pimcore.element.selector.quickSearchFacade');
+
+pimcore.element.selector.quickSearchFacade = new Class.create({
```

Comment:
lets combine that into one facade and add `showQuickSearch` and `hideQuickSearch` into `pimcore.element.selector.searchFacade`

Reply:
done 9d4d7d1f1318d3f930c83f785b5bba54bceba211